aten/src/ATen/native/mps/operations/Inverse.mm

```diff
@@ -24,6 +24,10 @@
     MPSStream* stream = getCurrentMPSStream();
     info.zero_();
 
+    if (A.numel() == 0) {
```

Comment:
Can you move this code along with `info.zero_()` to the beginning of this function? Because if `A.numel() == 0` , then the `A.to("cpu")` in the line 17 may cause problems:
` at::linalg_inv_ex_out(cpu_result, cpu_info, A.to("cpu"));`